lib/tasks/scripts/import_blocks_to_regions.rb

```diff
@@ -56,7 +56,8 @@ def find_or_create_region(name:, type:, parent:)
 
   def create_region_from(name:, type:, parent:)
     region = Region.new(name: name, type: type)
-    region.path = [parent.path, region.name_to_path_label].compact.join(".")
+    region.send(:set_slug) unless region.slug
```

Comment:
Don't think I understood why this is necessary. If FID [runs](https://github.com/norman/friendly_id/blob/master/lib/friendly_id/slugged.rb#L250) `set_slug` already as a `before_validation` why does it matter if it's set previously or not?

Reply:
Would we not have to do this in `path_label` as well?

It seems like you can't have a `path` as blank, but `path` depends on `slug` (which also can't be blank), so you'd have to set slug before you set the path, since the `set_path` method will blow during `before_validation` otherwise.

I don't know how to correctly solve this, but I find this dependency chain of path versus slug quite confusing.